cypress/e2e/smoke/P2P/chatFunctionalityActive.cy.js

```diff
@@ -0,0 +1,134 @@
+  cy.get('.chat-footer-icon-container').click()
+  cy.get('input[type="file"]').selectFile(
+    'cypress/fixtures/P2P/orderCompletion.png',
+    { force: true }
+  )
+  cy.findByPlaceholderText('Enter message')
+    .should('be.visible')
+    .type('Yes received')
+  cy.get('.chat-footer-icon-container').click()
+  cy.get('.chat-messages-item--outgoing').should('have.length', 2)
+  cy.get('.chat-messages-item-image').should('have.length', 3)
+}
+
+function buyerReplyChatToSellerValidation() {
+  cy.findByTestId('testid').click()
+  cy.findByText(
+    "Hello! This is where you can chat with the counterparty to confirm the order details.Note: In case of a dispute, we'll use this chat as a reference."
+  ).should('be.visible')
+  cy.findByText('Yes received').should('be.visible')
+  cy.get('.chat-messages-item-image').should('have.length', 3)
+  cy.get('.chat-messages-item--incoming').should('have.length', 2)
+  cy.get('.order-chat__messages-item-timestamp').should('have.length', 5)
+}
+
+function buyerInitialChatToSeller() {
+  cy.findByTestId('testid').click()
+  cy.findByText(
+    "Hello! This is where you can chat with the counterparty to confirm the order details.Note: In case of a dispute, we'll use this chat as a reference."
+  ).should('be.visible')
+
+  cy.get('.chat-footer-icon-container').click()
+  cy.get('input[type="file"]').selectFile(
+    'cypress/fixtures/P2P/orderCompletion.png',
+    { force: true }
+  )
+  cy.findByPlaceholderText('Enter message')
+    .should('be.visible')
+    .type('Hi I have paid the amount')
+  cy.get('.chat-footer-icon-container').click()
+  cy.get('.chat-messages-item--outgoing').should('have.length', 2)
+  cy.get('.order-chat__messages-item-timestamp').should('have.length', 2)
+  cy.findAllByTestId('dt_mobile_full_page_return_icon').eq(1).click()
+
+  cy.findByRole('button', { name: "I've paid" })
+    .should('not.be.disabled')
+    .click()
+}
+
+describe('QATEST-2791 Chat Functionality - Active', () => {
+  before(() => {
+    cy.clearAllSessionStorage()
+  })
+  beforeEach(() => {
+    if (isUser == true) {
+      loginWithNewUser('p2pChatActive1', false)
+    } else {
+      loginWithNewUser('p2pChatActive2', true)
+    }
+    cy.c_visitResponsive('/appstore/traders-hub', 'small'),
+      {
+        rateLimitCheck: true,
+      }
+  })
+
+  it('should login to the seller account and get the seller profile name', () => {
+    cy.c_navigateToP2P()
+    cy.findByText('My profile').should('be.visible').click()
+    cy.c_getProfileName().then((name) => {
+      nicknameAndAmount.seller = name
+    })
+  })
+
+  it('should login to buyer account , place a buy order and send a message with attachment on the chat', () => {
+    cy.c_navigateToP2P()
+    cy.findByText('My profile').should('be.visible').click()
+    placeBuyOrder(nicknameAndAmount.seller)
+    buyerInitialChatToSeller()
+    cy.c_uploadPOT('cypress/fixtures/P2P/orderCompletion.png')
```

Comment:
If we end the test on uploadPOT, cypress will perform this step and go to next it block. Hence, the POT will not be successfully uploaded.add an assertion after this to make sure that the POT is uploaded successfully. 
```suggestion
    cy.c_uploadPOT('cypress/fixtures/P2P/orderCompletion.png')
    cy.findByText('Waiting for the seller to confirm').should('be.visible')
```